tests/nfv/openvswitch_dpdk.pm

```diff
@@ -23,12 +24,14 @@ use utils;
 
 
 sub run {
+    my $dpdk_repo = "http://dpdk.org/git/dpdk";
+
     select_console 'root-console';
 
-    zypper_call('in openvswitch-switch', timeout => 200);
-    if (check_var('DPDK', 1)) {
-        zypper_call('in openvswitch-dpdk', timeout => 200);
-    }
+    zypper_call('in openvswitch-switch dpdk qemu git-core', timeout => 200);
```

Comment:
currently openvswitch_dpdk and vsperf_installation doing a lot of thing in common : 
1. both install some packages needed by "run_integration_tests" ( they even try to install same package - git-core ) 
2. both clone repositories needed by "run_integration_tests"
3. they run one after another 

keep all this in mind my question is - is there chance that they will run independently ? Or probably it make sense to combine them in single module with name "before_integration_tests" ?

Reply:
agreed with @jlausuch to do this in separate PR